internal/pkg/controllers/spod/bindata/spod.go

```diff
@@ -82,89 +88,212 @@ var Manifest = &appsv1.DaemonSet{
+							},
 						},
-					},
-					Resources: v1.ResourceRequirements{
-						Requests: v1.ResourceList{
-							v1.ResourceMemory:           resource.MustParse("64Mi"),
-							v1.ResourceCPU:              resource.MustParse("100m"),
-							v1.ResourceEphemeralStorage: resource.MustParse("50Mi"),
+						SecurityContext: &v1.SecurityContext{
+							AllowPrivilegeEscalation: &falsely,
+							ReadOnlyRootFilesystem:   &truly,
+							Capabilities: &v1.Capabilities{
+								Drop: []v1.Capability{"ALL"},
+							},
+							RunAsUser:  &userRootless,
+							RunAsGroup: &userRootless,
+							SELinuxOptions: &v1.SELinuxOptions{
+								// TODO(jaosorior): Use a more restricted selinux type
+								Type: "spc_t",
+							},
 						},
-						Limits: v1.ResourceList{
-							v1.ResourceMemory:           resource.MustParse("128Mi"),
-							v1.ResourceCPU:              resource.MustParse("300m"),
-							v1.ResourceEphemeralStorage: resource.MustParse("200Mi"),
+						Resources: v1.ResourceRequirements{
+							Requests: v1.ResourceList{
+								v1.ResourceMemory:           resource.MustParse("64Mi"),
+								v1.ResourceCPU:              resource.MustParse("100m"),
+								v1.ResourceEphemeralStorage: resource.MustParse("50Mi"),
+							},
+							Limits: v1.ResourceList{
+								v1.ResourceMemory:           resource.MustParse("128Mi"),
+								v1.ResourceCPU:              resource.MustParse("300m"),
+								v1.ResourceEphemeralStorage: resource.MustParse("200Mi"),
+							},
 						},
-					},
-					Env: []v1.EnvVar{
-						{
-							Name: config.NodeNameEnvKey,
-							ValueFrom: &v1.EnvVarSource{
-								FieldRef: &v1.ObjectFieldSelector{
-									FieldPath: "spec.nodeName",
+						Env: []v1.EnvVar{
+							{
+								Name: config.NodeNameEnvKey,
+								ValueFrom: &v1.EnvVarSource{
+									FieldRef: &v1.ObjectFieldSelector{
+										FieldPath: "spec.nodeName",
+									},
 								},
 							},
 						},
 					},
-				}},
+					{
+						Name:  "selinuxd",
+						Image: "quay.io/jhrozek/selinuxd",
+						Args: []string{
+							"daemon",
+							"--socket-path", SelinuxdSocketPath,
+							"--socket-uid", "0",
+							"--socket-gid", "2000",
+						},
+						ImagePullPolicy: v1.PullAlways,
+						VolumeMounts: []v1.VolumeMount{
+							{
+								Name:      "selinux-drop-dir",
+								MountPath: SelinuxDropDirectory,
+							},
+							{
+								Name:      "selinuxd-socket-volume",
+								MountPath: SelinuxdSocketDir,
+							},
+							{
+								Name:      "host-fsselinux-volume",
+								MountPath: "/sys/fs/selinux",
+							},
+							{
+								Name:      "host-etcselinux-volume",
+								MountPath: "/etc/selinux",
+							},
+							{
+								Name:      "host-varlibselinux-volume",
+								MountPath: "/var/lib/selinux",
+							},
+						},
+						SecurityContext: &v1.SecurityContext{
+							// TODO(jhrozek): If we wanted RO fs, we would need to mount
+							// the selinuxd db elsewhere than /var/run/selinuxd.db
```

Comment:
Let's open up a Github issue for this. We could either make that directory configurable, or just expect a directory inside of `/var/run`.

Reply:
I'll do that